assignment.js

```diff
@@ -65,6 +71,19 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let values = [3, 48, 29, 12, 18]
+console.log(`Values are ${values}`)
+let result = values[0]
+
+values.forEach(function(element, index){
```

Comment:
@kertcodes nice use of `.forEach()` to get this working. An alternative approach would be to call `.sort()` and `.reverse()` on the `values` array and then just get the first item

```
values.sort().reverse()[0]
```